src/renderer/components/common/QrCode/common/constants.ts

```diff
@@ -42,11 +42,11 @@ export const SIGNED_TRANSACTION_BULK = taggedUnion('SignaturesBulk', {
   4: ['V1', ['payload', array(MULTI_SIGNATURE)]],
 });
 
-const METADATA_PORTAL_URL = 'https://metadata.novasama.io/#/';
+const METADATA_PORTAL_METADATA_URL = 'https://metadata.novasama.io/?tab=1#/';
```

Comment:
Are you sure about `https://metadata.novasama.io/?tab=1#/`?
Maybe it's something like this - `https://metadata.novasama.io/#/?tab=1`?

Reply:
Yes, I'm sure. I've checked it. https://metadata.novasama.io/#/?tab=1/kusama does nothing.